spec/controllers/financial_transactions_controller_spec.rb

```diff
@@ -138,22 +122,4 @@
       end
     end
   end
-
-  describe 'PUT #update' do
-    it 'cannot find a route' do
-      ft = create :financial_transaction
-      expect do
-        put :update, params: { id: ft.to_param }
-      end.to raise_error AbstractController::ActionNotFound
-    end
-  end
-
-  describe 'DELETE #destroy' do
```

Comment:
I don't think you need to remove the destroy method as our permanent_records gem overrides the behavior. Instead of deleting the record when you call .destroy it adds a 'deleted_at' datetime column entry.

Reply:
We actually never had a destroy method so I figured we didn't want it in the first place.

Reply:
Apologies, yep you're correct to remove this if the method it is testing doesn't exist... but it just begs the question why was it here at all? I'm still wondering why we don't have a destroy method for financial transactions

Reply:
I'd like to think this ended up here after a bunch of copy-pasting + making relevant edits to make the tests pass

We don't delete Financial Transactions because we'd like to keep track of everything related to money so it doesn't make sense to just delete these records.  We could consider adding a functionality to 'deactivate' Financial Transactions, but I don't think that that adds any useful functionality.

Reply:
Well that was my point above, we already have permanent_records implemented so if we made a destroy method they wouldn't be deleted as permanent_records overrides that functionality. Making a destroy method would be effectively making a 'deactivate' method.

Reply:
I see.
I'd rather put that in a new issue and pull request then, based on what I read from the permanent_records documentation.  I think what is currently in this pull request is enough for the MVP. What do you think?

Reply:
Agreed. I'll make the issue now